hack/manifests/deployment.yaml

```diff
@@ -19,7 +21,7 @@ spec:
       - envFrom:
         - configMapRef:
             name: astro
-        image: quay.io/fairwinds/astro:v1.3.0
+        image: quay.io/fairwinds/astro:v1.4.0
```

Comment:
Neither of these tags exist (`v1.3.0` or `1.4.0`), I'm surprised it ever worked. Also I question the importance of the e2e tests that have a static tag. It will never actually test the changes made. Maybe we can put an environment variable into the script that runs.

Reply:
Oh nvm I see now that the pre script should change it:

`yq w -i hack/manifests/deployment.yaml spec.template.spec.containers[0].image "quay.io/fairwinds/astro:$CI_SHA1"`